bin/release/src/release.clj

```diff
@@ -20,10 +19,8 @@
 (def ^:private steps*
   (ordered-map/ordered-map
    :build-uberjar                       uberjar/build-uberjar!
-   :build-docker                        docker/build-docker-image!
-   :push-git-tags                       git-tags/push-tags!
    :upload-uberjar                      uberjar/upload-uberjar!
-   :push-docker-image                   docker/push-docker-image!
+   :push-git-tags                       git-tags/push-tags!
```

Comment:
The tag is pushed later, after the Uberjar is uploaded, so that `release.yml` can properly grab that Uberjar and package it for Dockerhub.